tutor/templates/build/openedx/Dockerfile

```diff
@@ -181,8 +181,14 @@ COPY --chown=app:app settings/lms/*.py ./lms/envs/tutor/
 COPY --chown=app:app settings/cms/*.py ./cms/envs/tutor/
 
 # Pull latest translations via atlas
-RUN atlas pull --repository='{{ ATLAS_REPOSITORY }}' --branch='{{ ATLAS_REVISION }}' {{ ATLAS_OPTIONS }} \
-    translations/edx-platform/conf/locale:conf/locale
+RUN ./manage.py lms --settings=tutor.i18n pull_plugin_translations --verbose --repository='{{ ATLAS_REPOSITORY }}' --revision='{{ ATLAS_REVISION }}' {{ ATLAS_OPTIONS }}
+RUN ./manage.py lms --settings=tutor.i18n pull_xblock_translations --repository='{{ ATLAS_REPOSITORY }}' --revision='{{ ATLAS_REVISION }}' {{ ATLAS_OPTIONS }}
+RUN atlas pull --repository='{{ ATLAS_REPOSITORY }}' --revision='{{ ATLAS_REVISION }}' {{ ATLAS_OPTIONS }} \
+    translations/edx-platform/conf/locale:conf/locale \
+    translations/studio-frontend/src/i18n/messages:conf/plugins-locale/studio-frontend
```

Comment:
A general question, how much of an impact will it have on the build image time/process? The pull is hitting the same repo again and again with different params it seems (linking with https://github.com/overhangio/tutor-mfe/pull/184). I am thinking from point of view of network consumption and its impact on overall image build process.

Reply:
Great questions @DawoudSheraz!

In short: It takes time, but it's well worth it in my opinion.

This checkout is a shallow checkout for a specific part of the directories, which relatively highly optimized [sparse-checkout `git clone`](https://github.com/openedx/openedx-atlas/blob/main/atlas#L470-L474).

I've ran the command on the `edx-platform` repo in my home internet and it took less than a minute in the `openedx/devstack`:

```
$ make OPENEDX_ATLAS_PULL=yes pull_translations
real	0m42.797s
user	0m0.025s
sys	0m0.025s
```` 

I presume similar build time with Tutor Dockerfile since the steps are relatively copied from each other.

Compared with the deprecated `tx pull` this is a much much slower process and gives stale translations every time. It also depends on a private 2U-controlled Jenkins instance.

OEP-58 explains the motivation behind this refactor:

 - https://docs.openedx.org/en/latest/developers/concepts/oep58.html

Reply:
Thanks for the explanation and a detailed context, much appreciated.